docs/using-oci-volume-provisioner.md

```diff
@@ -0,0 +1,173 @@
+# OCI Volume Provisioner
+
+The OCI Volume Provisioner enables [dynamic provisioning][1] of storage
+resources when running Kubernetes on Oracle Cloud Infrastructure (OCI). It uses
+the [OCI Flexvolume Driver][2] to bind storage resources to Kubernetes Nodes.
+The volume provisioner offers support for [Block Volumes][5] and File System
+Storage volumes.
+
+
+## Deployment
+
+The oci-volume-provisioner is provided as a Kubernetes Deployment.
+
+### Prerequisites
+
+  - Install the [OCI flexvolume driver][2] if provisioning Block Storage
+    volumes.
+
+
+### Configuration
+
+You will first need to create a cloud-provider configuration file tailored to
+your OCI account and Kubernetes cluster. Please see the provider configuration
+documentation [here][5].
+
+```bash
+$ kubectl create secret generic oci-volume-provisioner \
+  -n kube-system \
+  --from-file=config.yaml=cloud-provider.yaml
+```
+
+### OCI Permissions
+
+If authenticating with OCI via an IAM user please ensure it has the following
+privileges in the OCI API by creating a [policy][6] tied to a group or user.
+
+```
+Allow group <name> to manage volumes in compartment <compartment>
+Allow group <name> to manage file-systems in compartment <compartment>
+```
+
+## Deploy the OCI Volume Provisioner
```

Comment:
Not sure how to phrase this, but, maybe could also do with a little 'shout-out'  to the storage classes and that they are getting deployed here too? Feel free to ignore. Minor.